primitives/src/assets/market_assets.rs

```diff
@@ -27,58 +27,34 @@ use super::*;
 #[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
 #[derive(Clone, Copy, Debug, Decode, Eq, Encode, MaxEncodedLen, PartialEq, TypeInfo)]
 pub enum MarketAssetClass<MI: HasCompact + MaxEncodedLen> {
-    // All "Old" variants will be removed once the lazy migration from
-    // orml-tokens to pallet-assets is complete
     #[codec(index = 0)]
-    OldCategoricalOutcome(MI, CategoryIndex),
+    CategoricalOutcome(MI, CategoryIndex),
 
     #[codec(index = 1)]
-    OldScalarOutcome(MI, ScalarPosition),
+    ScalarOutcome(MI, ScalarPosition),
 
     #[codec(index = 3)]
-    OldPoolShare(PoolId),
+    PoolShare(PoolId),
```

Comment:
Just a general question: Wouldn't even renaming struct fields lead to storage migrations?

Reply:
Nope, they are simply encoded into `concat(index, type)`. You can see that types are properly encoded and decoded [within the asset scale_codec tests](https://github.com/zeitgeistpm/zeitgeist/blob/sea212-new-asset-system-part-5/primitives/src/assets/tests/scale_codec.rs)